### ISSUE TITLE
Localization site

### ISSUE THREAD

**Issue by KovalevArtem** [mozilla/ichnaea#1253]
https://location.services.mozilla.com/

It is sad that the site is only in English - I propose to translate.
I am ready to help in the translation into Russian.

After all, for this we can use the Pontoon platform?

**Comment by jwhitlock** [comment]
Thank you, @KovalevArtem. This is the first request I have seen to localize the website, and it is currently a low priority. If others are interested in translation or in translated content, please comment or vote up the issue by adding a 👍 reaction.

You are correct, we'd use Pontoon for translation, but there is a lot of work to get there. I think the steps would be:

* Investigate if we have string extraction tools for the chameleon template language.
* If we don't, convert the template language from chameleon to Jinja2, which does have widely-used tools at Mozilla
* Work with the Pontoon team to add a MLS project
* Add processes for exporting strings to Pontoon
* Add processes for importing vastest translations from Pontoon
* Add Russian
* Add UI for switching between translations

**Comment by jwhitlock** [comment]
There was a previous request, in issue #516, opened Mar 25, 2018:  

> In-case the frontend receives an update to the new Mozilla design in the future, it would be nice if the strings could also be saved to ftl files, so that the frontend can be localized by contributors at https://pontoon.mozilla.org/

Usually, the older bug wins, but in this case, I'm merging to this bug, where there has been more discussion.

My previous experience is with MDN, which has a [process for starting a new localization](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize/Starting_a_localization). It had already done the work of extracting strings for the UI, so adding a new translation wasn't a huge technical process. However, there are some criteria that we found useful to ensure the translation was successful:

* An active Mozilla localization community for the locale exists
* A designated localization leader for the locale has committed to leading the localizing effort.

Before we start the project, I'd also want the localization teams (for example, the [Russian team](https://pontoon.mozilla.org/ru/info/) and the [German team](https://pontoon.mozilla.org/de/info/)) to agree to support translation of a new website. I'd guess there would be between 50 and 250 strings once we convert the site.
